cmd/get-hardware-details/main.go

```diff
@@ -34,22 +36,34 @@ func main() {
 		InsecureSkipVerify: ironicInsecure,
 	}
 
-	inspector, err := clients.InspectorClient(opts.Endpoint, opts.AuthConfig, tlsConf)
+	endpoint := opts.Endpoint
+	parsedEndpoint, err := url.Parse(endpoint)
 	if err != nil {
-		fmt.Printf("could not get inspector client: %s", err)
+		fmt.Printf("invalid ironic endpoint: %s", err)
 		os.Exit(1)
 	}
 
-	introData := introspection.GetIntrospectionData(inspector, opts.NodeID)
+	if parsedEndpoint.Port() == "5050" {
```

Comment:
Big comment explaining what this is doing and when it can be removed wouldn't hurt.

Reply:
Okay, I'll follow-up with it.